src/StoreKit/SKAdNetwork.cs

```diff
@@ -0,0 +1,25 @@
+//
+// SKAdNetwork.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2018 Microsoft Corporation.
+//
+
+using System;
+using Foundation;
+using ObjCRuntime;
+
+#if TVOS && !XAMCORE_4_0
+namespace StoreKit {
+	[Obsolete ("Not usable from tvOS and will be removed in the future.")]
+	[Unavailable (PlatformName.TvOS)]
+	public class SKAdNetwork : NSObject {
+
+		[Obsolete ("Throws a 'NotSupportedException'.")]
+		[Unavailable (PlatformName.TvOS)]
+		public static void RegisterAppForAdNetworkAttribution () => throw new NotSupportedException ();
+	}
+}
+#endif
```

Comment:
minor (no need to update unless other changes are required). We generally create `{prefix}Compat.cs` files for those _hopefully_ short(er)-lived compatibility stubs. It should ease moving to a new profile (git rm) without messing to much with history. It also minimize the number of place to look for them.

Reply:
Ahh you are completely right, I forgot about it :( sorry about that.